app/templates/components/forms/orders/order-form.hbs

```diff
@@ -57,9 +57,22 @@
                 {{#if field.isLongText}}
                   {{widgets/forms/rich-text-editor value=(mut (get holder field.fieldIdentifier))
                                                    name=(if field.isRequired (concat field.fieldIdentifier '_required_' index) (concat field.fieldIdentifier '_' index))}}
+                {{else if (eq field.fieldIdentifier 'gender')}}
+                  {{#ui-dropdown class='search selection' selected=holder.gender onChange=(action (mut holder.gender)) as |execute mapper|}}
```

Comment:
use (get holder field.fieldIdentifier) for uniformity in code.

Reply:
@CosmicCoder96 Did the changes